health/writer/destination.go

```diff
@@ -24,11 +28,25 @@ type LogDestination struct {
 }
 
 // Push takes health data and builds a nice log message with it on info level
-func (l *LogDestination) Push(health *target.Health) error {
+func (l *LogDestination) Push(ctx context.Context, health *target.Health) error {
 	l.logTargetHealth(health)
 	return nil
 }
 
+// Register takes target data and builds a nice log message with it on info level
```

Comment:
Need to update comment?

Reply:
I believe that not. Register here is doing the same as Push, just outputs the data as a log message